### PR TITLE
Add security changes needed for docker

### PR DESCRIPTION
**Pull request by efeubiqube**

### CODE REVIEW

**Review by ArnaudFosse — APPROVED** [review]

File: doc-src/admin-guide/admin-guide-single.adoc [hunk 1]
```diff
@@ -45,3 +45,5 @@ include::{includedir}/support_and_troubleshooting.adoc[]
 
 include::{includedir}/opensource_technologies.adoc[]
 
+include::{includedir}/security_configuration.adoc[]
```
Comment (abr-ubiqube) [inline doc-src/admin-guide/admin-guide-single.adoc:48]: @efeubiqube 
this doc needs to be included after include::{includedir}/ha_installation.adoc[]

File: doc-src/admin-guide/security_configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
+= Security Configuration
+:doctype: book
```
Comment (abr-ubiqube) [inline doc-src/admin-guide/security_configuration.adoc:2]: to be removed

File: doc-src/admin-guide/security_configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
+= Security Configuration
+:doctype: book 
+
```
Comment (abr-ubiqube) [inline doc-src/admin-guide/security_configuration.adoc:3]: add 
ifdef::env-github,env-browser[:outfilesuffix: .adoc]

File: doc-src/admin-guide/security_configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
+= Security Configuration
```
Comment (abr-ubiqube) [inline doc-src/admin-guide/security_configuration.adoc:1]: change to "Security Configuration for HA Setup"

**Review by abr-ubiqube — CHANGES_REQUESTED** [review]

**Review by abr-ubiqube — APPROVED** [review]

**Pull request merged**